bin/cd.php

```diff
@@ -9,14 +9,14 @@
 use CentralDesktop\API\Workspaces\WorkspacesCommand;
 use Symfony\Component\Console\Application;
 use CentralDesktop\API\Auth\AuthCommand;
-use CentralDesktop\API\Pagination\PaginationCommand;
+use CentralDesktop\API\Times\TimesCommand;
 
 $application = new Application();
 $application->add(new AuthCommand());
 $application->add(new WorkspacesCommand());
 $application->add(new UsersCommand());
 $application->add(new UploadFileCommand());
-$application->add(new PaginationCommand());
+$application->add(new TimesCommand());
```

Comment:
/v1/workspaces is currently broken on prod and /v1/users doesn't have the latest pagination mechanism so I added a /v1/times which works as expected.